client/index.css

```diff
@@ -0,0 +1,77 @@
+@import "nav.css";
+@import "modal.css";
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@300&family=Quicksand&display=swap');
+
+:root {
+    --primary: #d87474;
+    --secondary: rgb(5,2,25);
+    --accent:   #0a7a89;
+    --text-primary: var(--primary);
+    --text-secondary: var(--secondary);
+    --text-accent:  var(--primary);
+    --font-primary: 'Montserrat', sans-serif;
+    --font-accent: 'Quicksand', sans-serif;;
+    --opacity: rgba(254, 247, 36, 0.837);
+}
+
+* { box-sizing: border-box; }
+
+html, body { 
+    height: 100%;
+    margin: 0;
+    font-family: var(--font-primary);
+}
+
+a { text-decoration: none; }
+
+body {
+    display: grid;
+    grid-template-rows: 5vh 90vh 5vh;
+    position: fixed;
+    width: 100vw;
+    text-align: center;
+}
+
+main {
+    padding: 10px 0;
+    width: 100vw;
+    max-height: 95vh;
+    margin: 0 auto;
+    grid-gap: 10px;
+    overflow-y: scroll;
+    overflow-x: hidden;
+    z-index: -1;
+    text-align: center;
+}
+
+footer {
+    background-color: var(--accent);
+    opacity: 0.3;
+    padding-top: 10px;
+}
+
+footer a { color: var(--text-accent); }
+
+.title { 
+    font-family: var(--font-accent);
+    grid-column: 1 / -1;
+    text-align: center;
+}
+
+.card {
+    object-fit: cover;
+    width: 50%;
+    height: 100px;
+    margin: 25px auto;
+    border: 2px solid grey;
+    padding: 25px;
+    background: var(--opacity);
+}
+
+.card:hover {
+    border: 2px solid var(--accent);
+    background: rgba(253, 253, 170, 0.7);
+}
+
+ul, ol, li { padding: 0; list-style: none; margin: 0; }
```

Comment:
Your website layout is cute and fun, loved it, if you saw us you'd think we're boring as hell.